lib/perl/Genome/Site/TGI/Extension/Sys.pm

```diff
@@ -128,6 +123,15 @@ sub _bsub_and_wait_for_completion__validate_args {
     return(\%special_args, %bsub_args);
 }
 
+sub _bsub_and_wait_for_completion__wait_on_jobs_with_lsf_dependency {
+    my ($class, $job_ids, $bsub_args) = @_;
+    local $ENV{LSF_NIOS_JOBSTATUS_INTERVAL} = 1;
+    $class->bsub(%$bsub_args,
+                 wait_for_completion => 1,
+                 depend_on => (join "&&", map { "ended($_)" } @$job_ids) ,
+                 cmd => "echo DONE");
+}
+
 sub _bsub_and_wait_for_completion__wait_on_jobs {
```

Comment:
If we're removing the part that sets up a socket, we should probably also remove this subroutine that uses the socket.